src/internal/kopia/wrapper.go

```diff
@@ -410,13 +410,16 @@ func (w Wrapper) RestoreMultipleItems(
 
 		ds, err := getItemStream(ctx, itemPath, snapshotRoot, bcounter)
 		if err != nil {
-			et.Add(err)
+			et.Add(clues.Stack(err).Label(fault.LabelForceNoBackupCreation))
```

Comment:
I don't think this is strictly necessary since it's mostly on the restore path

Reply:
StreamStore calls this to retrieve the backup details that it streams back when merging backups.  see: https://github.com/alcionai/corso/blob/main/src/internal/streamstore/streamstore.go#L115

Reply:
mmmmm I get that, but is there a way to catch the error closer to the details merge code? It seems odd to place it here and then ignore it for the restore path :/

Reply:
I don't think it's odd.  That's the nature of labels.  Consumers don't need to concern themselves with the complete set of values.  In the meantime, the labeling i/o matches here.  If we caught it upstream from the adder then we'd have to investigate added errors and append labels post-hoc.  Much messier, and much more error prone.

Reply:
eh sure, kind of an implementation detail I suppose